app/src/test/java/org/simple/clinic/summary/teleconsultation/status/TeleconsultStatusInitTest.kt

```diff
@@ -6,13 +6,17 @@ import com.spotify.mobius.test.InitSpec
 import com.spotify.mobius.test.InitSpec.assertThatFirst
 import org.junit.Test
 import org.simple.clinic.teleconsultlog.teleconsultrecord.TeleconsultStatus
+import java.util.UUID
 
 class TeleconsultStatusInitTest {
 
   @Test
   fun `when screen is created, then prefill teleconsult status if it exists`() {
     val initSpec = InitSpec(TeleconsultStatusInit())
-    val model = TeleconsultStatusModel.create(TeleconsultStatus.Yes)
+    val model = TeleconsultStatusModel.create(
+        teleconsultRecordId = UUID.fromString("60af7f4a-b61f-40ae-a2b9-9f5273a70d4e"),
+        teleconsultStatus = TeleconsultStatus.Yes
```

Comment:
Creating a model in the tests should follow the process they do in production. In production, we will never have a teleconsult status ready when creating the model and we'll end up constructing it via the events. We should follow the same flow here.

```kotlin
val model = TeleconsultStatusModel
  .create(UUID.fromString())
  .teleconsultStatusLoaded()
```

Reply:
If user selects the still waiting status, won't we have to reselect it next time the sheet is shown? 🤔 . That was the only reason why I included default teleconsult status option, or else we can remove prefill effect as well.

Reply:
Isn't that taken care of by the prefill status effect? When the effect is triggered, it prefills the UI which triggers a checked change event which then updates the model again.

Reply:
NVM, turns out we don't have to prefill the status in sheet. Confirmed with Pragati, sorry 😅 

Will remove the prefill effect as well default status in the factory method.

Reply:
Correct me if I am wrong, State restoration should automatically work with the RadioGroup so we don't manually have to prefill it in those scenarios as well right?

Reply:
Yup, that should be handled by the system